packages/frontend/src/components/Game/GameBoard/index.test.jsx

```diff
@@ -68,4 +68,49 @@ describe('GameBoard component', () => {
     expect(tile.photo(tiles[1].photo.alt).query()).not.toBeInTheDocument();
     expect(tile.photo(tiles[5].photo.alt).query()).not.toBeInTheDocument();
   });
+
+  it('will not flip a tile to face down with second click', async () => {
+    const tiles = createTilesFromPhotos(mockPhotos, { shuffle: false });
+
+    const flippedTileState = produce(baseState.game, (draft) => {
+      draft.tiles = tiles;
+    });
+
+    const state = { ...baseState, game: flippedTileState };
+
+    const { user, screen } = setupTests(GameBoard, { state });
+
+    expect(screen.getByTestId('tile-0')).toHaveClass('faceDown');
+
+    await user.click(screen.getByTestId('tile-0'));
+
+    act(() => jest.advanceTimersByTime(3000));
+
+    await user.click(screen.getByTestId('tile-0'));
+
+    act(() => jest.advanceTimersByTime(1000));
+
+    expect(screen.getByTestId('tile-0')).toHaveClass('faceUp');
+  });
+
+  it('will not flip a tile if two others are flipped', async () => {
+    const tiles = createTilesFromPhotos(mockPhotos, { shuffle: false });
+
+    const flipped = [tiles[3], tiles[9]];
+
+    const twoFlippedGameState = produce(baseState.game, (draft) => {
+      draft.tiles = tiles;
+      draft.flipped = flipped;
+    });
+
+    const state = { ...baseState, game: twoFlippedGameState };
+
+    const { user, screen } = setupTests(GameBoard, { state });
+
+    await user.click(screen.getByTestId('tile-0'));
+
+    act(() => jest.advanceTimersByTime(1000));
+
+    expect(screen.getByTestId('tile-0')).not.toHaveClass('faceUp');
```

Comment:
This test could also assert that the tile that's clicked is not face up *before* the click.

I'm not sure it's necessary to advance the timers here either.